Food/minectl.lua

```diff
@@ -0,0 +1,76 @@
+local name = "minectl"
+local version = "0.10.0"
+
+food = {
+    name = name,
+    description = "minectl 🗺️ is a cli for creating Minecraft (java or bedrock) server on different cloud provider.",
+    license = "Apache License 2.0",
```

Comment:
Please use "Apache-2.0" as per the SPDX specification: https://spdx.org/licenses/